data/adapters/gtex_sqtl_adapter.py

```diff
@@ -103,12 +103,13 @@ def process_file(self):
                                     'chr': variant_id_ls[0],
                                     'biological_context': self.ontology_term_mapping.get(filename_biological_context) or biological_context,
                                     'sqrt_maf': to_float(line_ls[5]),
+                                    'p_value': pvalue,
                                     'log10pvalue': log_pvalue,
                                     'pval_nominal_threshold': to_float(line_ls[9]),
                                     'min_pval_nominal': to_float(line_ls[10]),
-                                    'slope': to_float(line_ls[7]),
+                                    'effect_size': to_float(line_ls[7]),
                                     'slope_se': to_float(line_ls[8]),
```

Comment:
Would it make sense to also change `slope_se` to `effect_size_se`?

Reply:
Yes that makes sense, I can update it together with the `source_url` change in another PR, since the reloading will take a while.